examples/vllm_v1/README.md

```
@@ -41,9 +41,15 @@ git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
 
 Or install vLLM manually:
 
-```
-uv pip install vllm==0.9.1
-```
+'''bash
+export VLLM_REF=3c545c0c3b98ee642373a308197d750d0e449403
+git clone https://github.com/vllm-project/vllm.git
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)